code/game/objects/structures/crates_lockers/closets/secure/bsdroppod.dm

```diff
@@ -3,7 +3,7 @@
 
 //------------------------------------BLUESPACE DROP POD-------------------------------------//
 /obj/structure/closet/bsdroppod
-	name = "Bluespace Drop Pod"

```

Comment:
you mean /obj/structure/closet/sdroppod surely?

Reply:
no point in a repath